examples/js/renderers/CSS2DRenderer.js

```diff
@@ -161,8 +161,8 @@ THREE.CSS2DRenderer = function () {
 	};
 
 	this.render = function ( scene, camera ) {
-
-		scene.updateMatrixWorld();
+		
+		if (scene.autoUpdate === true) { scene.updateMatrixWorld(); }
```

Comment:
Please write this line like so:
```js
if ( scene.autoUpdate === true ) scene.updateMatrixWorld();
```

Reply:
Ok, no problem.
Have updated the pull request - would you mind double checking it's been updated? Fairly new to this.